frontend/assets/tailwind.css

```diff
@@ -40,4 +43,7 @@
   html::-webkit-scrollbar-thumb:hover {
     background-color: #a0a0a5;
   }
+  label.vue-treeselect__label {
+    color: pink;
```

Comment:
Pink?

Reply:
That was a test how its work or it would even change something, looks like i forgot to remove this part.